MaraBot/Core/PresetValidation.cs

```diff
@@ -92,15 +93,46 @@ public static List<string> ValidateOptions(in Preset preset, IReadOnlyDictionary
                 if (pairMode != mode && pairMode != Mode.General)
                     errors.Add($"{kValidationInfoPrefix} '{pair.Key}' belongs to the {Option.ModeToPrettyString(pairMode)} mode, but the selected mode is {Option.ModeToPrettyString(mode)}.");
 
-                // Skip if value doesn't exist
-                foreach (var o in allOptions[pair.Key].Type == OptionType.List
-                        ? Option.ParseList(pair.Value) : new List<string> { pair.Value })
-                    if (!allOptions[pair.Key].Values.ContainsKey(o))
+                // General enum validation
+                if (option is EnumOption)
+                {
+                    var enumOption = option as EnumOption;
+
+                    // Skip if value doesn't exist
+                    foreach (var o in option.List ? Option.ParseList(pair.Value) : new List<string> { pair.Value })
+                        if (!enumOption.Values.ContainsKey(o))
+                        {
+                            errors.Add($"{kValidationErrorPrefix} '{pair.Key}' has no known value '{o}'");
+                            optionsCopy.Remove(pair.Key); // Make sure we don't validate on unknown values
+                            continue; // Check if there are more unknown values, so that we can show those errors too
```

Comment:
this continue statement won't do anything in this case.

Reply:
The `continue` statement got a bit obsoleted due to code changes, but I kept it in because of the comment right after. This makes it explicit that we are to loop over *all* values, even if there's malformed ones.

If you want, I can try a completely different approach to the for-loop that removes any need for the explicit `continue`.

Reply:
I don't mind keeping the `continue` statements if you prefer it this way :)